packages/host/tests/acceptance/code-submode/spec-test.gts

```diff
@@ -351,6 +353,8 @@ module('Acceptance | Spec preview', function (hooks) {
     assert
       .dom('[data-test-spec-selector-item-path]')
       .hasText('pet-entry-2.json');
+    assert.dom('[data-test-view-spec-instance]').exists();
+    await click('[data-test-view-spec-instance]');
```

Comment:
You have to have another test that clicks on the button and assert that you have the spec instance in the RHS